### PR TITLE
Implement classes and objects in Ruby

### PR DESCRIPTION
**Pull request by Mucyosoda**
Implement classes and objects in Ruby
Create class Person
Create class Student
Create class Teacher

### CODE REVIEW

File: teacher.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require_relative('./person')
+
+class Teacher < Person
+  def can_use_services?
+    true
+  end
+end
```
Comment (hiromataba) [inline teacher.rb]: - [x] Your `Teacher` class should have a constructor that extends the parent's constructor by adding @specialization and a parameter for it, [Check out this lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) to learn more about constructors, Your constructor will look something like this :point_down: 

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi :wave: :wave: @Mucyosoda , 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :medal_sports: 
- Nice PR title and description :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

## Required Changes ♻️

- [ ] To make your project more professional you need to update your README file, Please have a look at [This template](https://github.com/microverseinc/readme-template) and update your README file

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: teacher.rb (hunk 1 above)
Comment (Mucyosoda) [reply teacher.rb]: Thank you

**Review by Emmyn5600 — APPROVED** [review]
Hi @Mucyosoda 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [ ] N/A

#### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!:clap::clap::clap:
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
#### _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**